isis/src/base/objs/Spice/Spice.cpp

```diff
@@ -1157,7 +1170,7 @@ namespace Isis {
     // Read from PvlObject that is our naif keywords
     QVariant result;
 
-    if (m_naifKeywords->hasKeyword(key) && !m_usingNaif) {
+    if (m_naifKeywords->hasKeyword(key) && m_usingAle) {
```

Comment:
Why is this changed? We still want to be able to read from the NaifKeywords object even when we didn't use ALE.

Reply:
changed it to be more explicit.